modern-gpus/amd-gpu.md

```diff
@@ -35,13 +37,20 @@ Supported Cards:
 * RX 6900 XT
   * The XTXH variant (Device ID: `0x73AF`) is supported with WhateverGreen 1.5.2 and spoofing `device-id` to `0x73BF`.
 
-Note: Some Navi 21 cards currently require the boot argument `agdpmod=pikera` to get a display out.
+Note: Most Navi 21 cards currently require the boot argument `agdpmod=pikera` to get a display out.
 
 Needed kexts:
 
 * [Lilu.kext](https://github.com/acidanthera/Lilu/releases)
 * [WhateverGreen.kext](https://github.com/acidanthera/WhateverGreen/releases)
 
+<details>
+  <summary>Navi 21 and 23 series without WhateverGreen</summary>
+  
+ On macOS Ventura, by setting iMacPro1,1 or MacPro7,1 SMBIOS you can use Navi 21 and 23 cards without WhateverGreen or `agdpmod=pikera` in boot args.
+  
+</details>
```

Comment:
```suggestion
```
This works for other macOS versions and other cards too. I did this with my GTX 1080 in High Sierra.
The main downside is that you'll be missing other patches, especially on GCN cards. It's probably best to not advise this. This really isn't the right place for something like this anyways.

In the future, dropdowns should use
```md
::: details <name>

<body>

:::
```

Reply:
Thanks. Noted the details block, I thought that spoiler github block was ok. I’ll review your comments later.